azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/AbstractAzureMojo.java

```diff
@@ -257,43 +232,20 @@ public String getPluginVersion() {
         return plugin.getVersion();
     }
 
-    @Override
     public String getUserAgent() {
         return isAllowTelemetry() ? String.format("%s/%s %s:%s %s:%s", getPluginName(), getPluginVersion(),
                 INSTALLATION_ID_KEY, getInstallationId(), SESSION_ID_KEY, getSessionId())
                 : String.format("%s/%s", getPluginName(), getPluginVersion());
     }
 
-    @Override
     public String getHttpProxyHost() {
         return httpProxyHost;
```

Comment:
getter

Reply:
this PR is not for this kind of refactor